governance/voting-committees/dvc-1/strategies.json

```diff
@@ -0,0 +1,10 @@
+{
+    "risk": {
+        "definition": "strategies/risk.md",
+        "delegates": ["0x7Ddb50A5b15AeA7e7cf9aC8E55a7F9fd9d05ecc6"]
```

Comment:
It should be the same, they are delegate contracts.

Reply:
True. I thought there will be Delegate contracts (as delegate profiles-ish) and their respective delegate-strategy contracts.

Reply:
> True. I thought there will be Delegate contracts (as delegate profiles-ish) and their respective delegate-strategy contracts.

I know we discussed this but I think we're going to avoid contracts as profiles for now and handle "profiles" and contract ownership relationships via metadata since it would require a larger contract factory refactor